index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width"> <!-- чтобы @media работал в хроме при нажатом Toggle Device Toolbar-->
+    <title>Responsive PSD Template</title>
+    <link href="css/style.css" rel="stylesheet">
+    <link href="http://allfont.ru/allfont.css?fonts=montserrat" rel="stylesheet" type="text/css" />
+</head>
+<body>
+    <div id="mySidenav" class="default-hidden sidenav">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+        <a href="#">PSD Websites</a>
+        <a href="#">UI Kits</a>
+        <a href="#">Premium PSD Files</a>
+        <a href="#">About</a>
+        <a href="#">Blog</a>
+    </div>
+
+    <div class="default-hidden menu-nav">
+        <div class="nav-text">Navigation</div>
+        <div class="menu-btn" onclick="openNav()">
+        </div>
+    </div>
+    <div class="menu default-visible">
+            <a href="#">PSD Websites</a>
+            <a href="#">UI Kits</a>
+            <a href="#">Premium PSD Files</a>
+            <a href="#">About</a>
+            <a href="#">Blog</a>
+        <span class="default-hidden">Navigation</span>
+    </div>
+    <div class="navigation default-hidden">
+
+    </div>
+
+    <div class="ad">
+        <div class="container">
+            <div class="text">Awesome Free PSD templates & Resources</div>
+            <div class="link">
+                <a href="#">www.BlazRogar.com</a>
+                <img src="img/arrow.png">
+            </div>
+        </div>
+    </div>
+    <div class="text_box">
+        <div class="quote">I'm like the Robin Hood of PSD files.</div>
+        <div class="btn"></div>
+    </div>
+
+    <div class="blocks">
+        <div class="arrow"></div>
+        <div><img src="img/bridge.png"></div>
+        <div><img src="img/woman.png"></div>
+    </div>
+
+    <div class="info">
```

Comment:
Semantically, it could be replaced with section and couple of articles

Reply:
I fixed all the issues, so what should I do next? Show my code on the lesson or create new pull request?

Reply:
Firstly push your changes to this PR please. Then I'll perform another code review. And see you on classes because we still need a face-to-face conversation to mark your task as approved and close this PR